dev/devicelab/lib/framework/cocoon.dart

```diff
@@ -73,15 +100,59 @@ class Cocoon {
       print('${rec.level.name}: ${rec.time}: ${rec.message}');
     });
 
-    final Map<String, dynamic> status = <String, dynamic>{
+    final Map<String, dynamic> updateRequest = _constructUpdateRequest(
+      gitBranch: gitBranch,
+      builderName: builderName,
+      result: result,
+    );
+    final Map<String, dynamic> response = await _sendCocoonRequest('update-task-status', updateRequest);
+    if (response['Name'] != null) {
+      logger.info('Updated Cocoon with results from this task');
+    } else {
+      logger.info(response);
+      logger.severe('Failed to updated Cocoon with results from this task');
+    }
```

Comment:
Nit: make this block of code a sub function so that both `sendResultsPath` and `sendTaskResult` can share.

Reply:
Done